### DISCUSSION TITLE
What's the principal of how it works?

### DISCUSSION THREAD

**Discussion by brianp** [new92/igfi#22] [Q&A] [answered]
I don't really get it. I was reviewing the script and it looks like all it really does is follow a pre-defined list of users, and then unfollow them again.

How does following and unfollowing the same accounts increase followers to a specific account?

**Comment by new92** [comment, marked as answer]
Hey Brian ! Let me explain it to you. As you may know on Instagram when you see the followers of a user the ones on the top are the new ones (it works like a stack). And there are users on Instagram which follow the new followers of celebrities. So what IGFI is doing is to put your account on the top of the followers of celebrities and let the users follow you. That's basically the philosophy of how it works and it's basically an automated operation so the user don't have to do it manually. Hope this helped ! If you have any other questions or want further information don't hesitate to contact me !

**Reply by SergeyOrl0v** [reply]
Sorry, will it unfollow celebrities after..?

**Reply by new92** [reply]
Yes of course ! 90% of the time the script will unfollow the users it followed. I'm working on this feature in order to make 100% sure that the users followed will be also unfollowed after the process.

**Reply by SergeyOrl0v** [reply]
Thank you so much!
